bindings/python/pymongoarrow/builders.pyi

```diff
@@ -13,27 +13,26 @@
 # limitations under the License.
 
 
-cdef class _BuilderBase:
+cdef class _ArrayBuilderBase:
     def append_values(self, values):
         for value in values:
             self.append(value)
 
-    @property
-    def null_count(self):
-        return self.builder.get().null_count()
-
-    def __len__(self):
-        return self.builder.get().length()
-
```

Comment:
Why was this required?

Reply:
There are some weird errors that crop up when you try to use C-namespace things from a parent class method in a child `cdef class`. The easiest workaround was to simply equip each `cdef class` with its own instance of this method.

Reply:
Thanks.